boards/arm/cy8cproto_062_4343w/cy8cproto_062_4343w_defconfig

```diff
@@ -29,3 +29,9 @@ CONFIG_GPIO=y
 
 # Enable clock controller
 CONFIG_CLOCK_CONTROL=y
+
+# Main Stack Size
+CONFIG_MAIN_STACK_SIZE=2048
+
+# Heap Pool Size
+CONFIG_HEAP_MEM_POOL_SIZE=98304
```

Comment:
This seems rather large, is there a reason? If so it should probably be mentioned here